packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/vue/nutrition.md

```diff
@@ -0,0 +1,18 @@
+This guide walks you through building a password sign-in Vue.js app that uses the Okta Auth JS and Vue.js libraries for Okta [embedded authentication](/docs/concepts/redirect-vs-embedded/#embedded-authentication).
+
+---
+
+**Learning outcomes**
+
+* Understand the sequence of steps required to implement Okta [embedded authentication](/docs/concepts/redirect-vs-embedded/#embedded-authentication) with Okta Auth JS and Vue.js libraries for your Vue.js app.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup/oie-preview.html)
+* [Okta Auth SDK](https://github.com/okta/okta-auth-js) (`@okta/okta-auth-js`)
+* [Okta Vue.js SDK](https://github.com/okta/okta-vue) (`@okta/okta-vue`)
+* [Vue CLI](https://cli.vuejs.org/guide/installation.html)
+
+    > **Note**: Use the latest version of the Okta libraries for your Vue.js app. <!--This guide was written for `@okta/okta-vue@5.1.1` and `@okta/okta-auth-js@6`. -->
+
+---
```

Comment:
Just a comment here. In the Nutrition facts snippet and the intro snippet (that appears directly after), we have 3 references to the Embedded authentication concept page. I suggest we remove the 2nd and 3rd references (3rd in intro) to avoid excessive links.

Reply:
I'll remove the first 2 links. Thanks!